src/main/flight/mixer.c

```diff
@@ -566,10 +580,95 @@ static void airplaneMixer(void)
 }
 #endif
 
+#ifdef USE_SERVOS
+uint8_t hasTiltingMotor(){
+	return currentMixerMode == MIXER_QUADX_TILT || currentMixerMode == MIXER_OCTOX_TILT;
+}
+
+/*
+ * return a float in range [-PI/2:+PI/2] witch represent the actual servo inclination wanted
+ */
+float getTiltServoAngle(void) {
+    float userInput = 0;
```

Comment:
'get' is reserved for getter/setters, prefer verbs like 'calculate', 'build', 'make' etc.